phpunit-test-reporter.php

```diff
@@ -93,3 +93,35 @@ function ptr_get_template_part( $template, $vars = array() ) {
 	include $full_path;
 	return ob_get_clean();
 }
+
+/**
+ * Counts the number of failing or passing test reports for a revision.
+ *
+ * @param int  $revision_parent_id The current revision's post ID.
+ * @param bool $failures           Whether to count failures. Default is false (count successful reports).
+ *
+ * @return int The number of reports for the given revision that either passed or failed.
+ */
+function ptr_count_test_results( $revision_parent_id, $failures = false ) {
```

Comment:
Maybe instead of a boolean `$failures` make this a `$status = 'passed' | 'failed'` field. 

More future proof and `ptr_count_test_results( $revision->ID,' failed' );` is easier to grasp than `ptr_count_test_results( $revision->ID, true )`